Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/CommonComponents/AdditionalFieldsWindow.tsx

```diff
@@ -69,9 +129,23 @@ function AdditionalFieldsWindow(props: IProps): JSX.Element {
                             <button
                                 className={"btn btn-primary" + ((changes.length === 0 || invalidChanges.length > 0) ? ' disabled' : '')}
                                 onClick={() => {
-                                    if (changes.length > 0 && invalidChanges.length === 0 && saveCallback.current.getHandle != null)
-                                        saveCallback.current.getHandle();
-                                }}
+                                    if (changes.length > 0 && invalidChanges.length === 0) {
+                                            setState('loading');
+                                            $.ajax({
+                                                type: "PATCH",
+                                                url: `${homePath}api/SystemCenter/AdditionalFieldValue/Array`,
+                                                contentType: "application/json; charset=utf-8",
+                                                data: JSON.stringify(changes),
+                                                dataType: 'json',
+                                                cache: true,
+                                                async: true
+                                            }).then(() => {
+                                                setChanges([]);
+                                                setState('changed');
```

Comment:
Can we also set the appropriate note slice as changed here based on the type prop? The new note if the setting is active doesn't load until refresh.

Reply:
Good catch